source/D2CommonDefinitions/include/D2PacketDef.h

```diff
@@ -296,14 +296,19 @@ struct D2GSPacketClt31		//size of 0x09
 	int32_t unk0x01[2];			//0x01
 };
 
+enum D2TransactionMode : uint32_t {
+    TRANSACTIONMODE_BUY = 0,
+    TRANSACTIONMODE_GAMBLE = 2,
+    TRANSACTIONMODE_FILL = 0x80000000
+};
```

Comment:
Not sure if this should live here.
Also, I guess we don't use enum types in packets to avoid having size mismatches?